lib/models/BucketInfo.ts

```diff
@@ -102,7 +113,7 @@ export default class BucketInfo {
     _azureInfo: any | null;
     _ingestion: { status: 'enabled' | 'disabled' } | null;
     _capabilities?: Capabilities;
-    _quotaMax: number | 0;
+    _quotaMax: bigint;
```

Comment:
quotaMax may not be set?

```suggestion
    _quotaMax?: bigint;
```

Reply:
The quota is always set, with a default to 0 (means no quota)

Reply:
it is true even in case of "upgrade", i.e. if the bucket was created before quotas were introduced? (and so mongo may not have the field)
i.e. is this structure what we have in memory (where indeed it may also be set), or what we have in mongo (where it may be missing)

Reply:
I mean, the constructor accepts an empty / void / undefined / null quota, but will always default to 0 in the class. So whatever the starting state: either we have that quota already, either it will be set to 0 at the next update.

So yes, this is fully backward compatible this way.